api/service.py

```
@@ -710,276 +834,91 @@ async def get_anomaly_history(
-    try:
-        # Get basic stats
-        training_data_count = len(predictive_engine.training_data)
 
-        return {
-            "status": "active",
-            "training_data_points": training_data_count,
-            "models_trained": len(predictive_engine.models),
-            "last_prediction": getattr(predictive_engine, '_last_prediction_time', None)
-        }
+@app.post("/api/v1/auth/apikeys", response_model=APIKeyCreateResponse)
+async def create_api_key(request: APIKeyCreateRequest, current_user: User = Depends(get_current_user)):
```

Ruff (0.14.10)

Do not perform function call `Depends` in argument defaults; instead, perform the call within the function, or read the default from a module-level singleton variable

(B008)